wpilibNewCommands/src/test/native/cpp/frc2/command/DefaultCommandTest.cpp

```diff
@@ -15,9 +16,9 @@ TEST_F(DefaultCommandTest, DefaultCommandSchedule) {
 
   TestSubsystem subsystem;
 
-  RunCommand command1([] {}, {&subsystem});
+  auto command = cmd::Run([] {}, {&subsystem});
```

Comment:
```suggestion
  auto command = cmd::Idle({&subsystem});
```
Same for the rest of the file.